src/System.Private.ServiceModel/src/System/ServiceModel/Description/XmlSerializerOperationBehavior.cs

```diff
@@ -128,6 +128,23 @@ void IOperationBehavior.AddBindingParameters(OperationDescription description, B
 
         void IOperationBehavior.ApplyDispatchBehavior(OperationDescription description, DispatchOperation dispatch)
         {
+            if (description == null)
+                throw DiagnosticUtility.ExceptionUtility.ThrowHelperArgumentNull("description");
+
+            if (dispatch == null)
+                throw DiagnosticUtility.ExceptionUtility.ThrowHelperArgumentNull("dispatch");
+
+            if (dispatch.Formatter == null)
+            {
+                dispatch.Formatter = (IDispatchMessageFormatter)CreateFormatter();
+                dispatch.DeserializeRequest = _reflector.RequestRequiresSerialization;
+                dispatch.SerializeReply = _reflector.ReplyRequiresSerialization;
+            }
+
+            if (_reflector.Attribute.SupportFaults && !dispatch.IsFaultFormatterSetExplicit)
```

Comment:
Thanks @khdang -- just a question.  What does it mean when you say "the wrapper is not used at all".  Does that mean we are not currently using that code path or that it is impossible to execute that code path in NET Core?